contents/handbook/people/team-structure/people.md

```diff
@@ -8,28 +8,41 @@ hideAnchor: true
 ## People
 
 - [Eltje Lange](/handbook/company/team#eltje-lange-people-and-talent)
+- [Recruitment and Operations Coordinator](https://apply.workable.com/posthog/j/554EC800BE/) (currently hiring!)
 
 ## Mission
 
-Make PostHog the best place everyone here has ever worked.
+Make PostHog the best place everyone here has ever worked. Our goal is to create a world-class (remote) culture, with a strong sense of impact, autonomy and career development.
 
 ## Responsibilities
 
-- Each hire should be unquestionably awesome.
-- Create an inclusive and diverse culture.
-- Once the above are met, hit our hiring goals.
-- Keep the candidate pipeline organized so we don't drop the ball on anyone. 
+Our people team works across talent, people and culture: 
+
+- We attract, engage and hire top talent from around the world, whilst ensuring an outstanding candidate and hiring manager experience.
+- Building a [diverse and inclusive culture](/handbook/company/diversity) is at the heart of everything we do. 
+- We support our team throughout the entire employee lifecycle - from making an offer, to onboarding and career development over time, to parental leave and eventually parting ways. 
+- We create light-touch initiatives and processes that allow PostHog to act fast (while being compliant to local legislation) and [iterate](/handbook/company/culture#iteration) continuously. 
+- In the people team, we live and breathe our [culture](/handbook/company/culture) and [values](/handbook/company/values) and constantly work to make PostHog an even better place to work.
 
 ## Customer
 
-Every other small team.
+All small teams as well as current, future and past candidates. 
 
 ## Output metrics
```

Comment:
@eltjehelene I cut these down to the 5 absolute core metrics that we always have at our fingertips (generally with metrics I have a strong preference for a small number that we know really well). 

Maybe worth adding how we measure each? e.g. team engagement - _quarterly engagement survey_?